apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/sprint-zero/creating-a-site.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Creating a Site | Sitecore Accelerate for partners'
+description: 'How to create your Site and Site Collection in XM Cloud'
+area: ['accelerate']
+hasSubPageNav: true
+hasInPageNav: false
+---
+
+## Problem
+
+I need to create a new Site, what is the best way to do it? Should it be in a Site Collection? Should I create a new site collection or put it in an existing one?
+
+## Solution
+
+When creating a Site in XM Cloud, a site collection is always required. Then Sites live in a site collection. You can have as many site collections as needed and a site collection can have many Sites.
+
+### Creating a Site Collection
+
+Before we create a site collection, we need to create an SXA Module for our templates and rendering items to live in. It is important to create the Headless Module before you create the site collection, once the site collection is created, it is not possible to create a `Headless Module`.
+
+In your XM Cloud instance, load up Content Editor and navigate to `/sitecore/System/Project`. Right-click and insert a new `Headless Module`. Make sure to name the module with the same name that you will use for your `Site Collection`.
+
+<img src="/images/learn/accelerate/xm-cloud/create-a-site-1.png" alt="Creata a new headless module"/>
+
+Make sure all the System areas and the scaffolding actions are selected. Click Proceed.
+
+Once the creation wizard has completed, navigate to `/sitecore/Content`. Right-click and insert a new `Headless Site Collection`. Make sure that you give the site collection the same name as the module we just created:
+
+<img src="/images/learn/accelerate/xm-cloud/create-a-site-2.png" alt="Creata a new headless site collection"/>
+
+Make sure to tick all the required modules and include the SXA Module created in the previous step.
+
+Click OK and wait for the site collection creation wizard to complete.
+
+You now have a site collection linked to an SXA module in preparation for component cloning.
+
+### Creating a new Site
+
+Right-click the site collection and insert a new `Site`. Make sure that in the `Modules` tab, the `Project` layer module created earlier is selected:
+
+<img src="/images/learn/accelerate/xm-cloud/create-a-site-3.png" alt="Creata a new headless site"/>
+
+Click OK and wait for the site creation wizard to complete.
+
+## Discussion
+
+### Do I need a new Site Collection?
+
+All sites must be placed within a Site Collection. Site Collections are a way to group multiple sites either need to share resources or are authored by a single group. With XM Cloud’s multiple site collections architecture, you can provide each site collection a dedicated share of the sitecore instance, including data templates, configuration, user management, and non-functional properties.
```

Comment:
...multiple sites that* either need to share...

Capitalize Sitecore*

Reply:
Fixed